components/mdz/Makefile

```
@@ -37,6 +37,10 @@ get-perfsprint-deps:
 perfsprint: get-perfsprint-deps
 	perfsprint ./...
 
+.PHONY: gotext
```

```suggestion
.PHONY: gotest
```
Please, fix the name for Makefile .PHONY: gotest.